File: src/main/java/cc/aliza/production/holiday/entity/Help.java
package cc.aliza.production.holiday.entity;

import com.bugull.mongo.SimpleEntity;
import com.bugull.mongo.annotations.Entity;

/**
 * Created by Jing on 14-2-17.
 */
@Entity
public class Help extends SimpleEntity {

    // 标题
    private String title;

    // 内容
    private String content;

    // 分类
    private String tag;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public String getTag() {
        return tag;
    }

    public void setTag(String tag) {
        this.tag = tag;
    }

}

File: src/main/webapp/resource/js/manage/system/help.js
/**
 * Created by Jing on 13-12-12.
 */

define(function (require, exports) {
    var $ = require('jquery');

    var base = $('body').data('base');

    $('.selectTag').click(function (e) {
        e.preventDefault();
        $('.tag').text($(this).text());
    });

    $('.save').click(function () {
        if ($('.tag').text() == '请选择') {
            return;
        }
        if (!$('.title').val()) {
            return;
        }
        $.ajax({
            url: base + '/manage/system/help/save',
            type: "POST",
            data: {
                id: $('[data-help]').data('help'),
                tag: $('.tag').text(),
                title: $('.title').val(),
                content: UE.getEditor('edit').getContent()
            },
            success: function (res) {
                if (res.success) {
                    location.href = base + '/manage/system/help';
                }
            }
        });
    });

    if ($('[data-help]').data('help')) {
        UE.getEditor('edit').setContent($('#content').html());
    }
});